CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,132 @@
+
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in our
```

Comment:
What are "members", "contributors", and "leaders"?

Reply:
The ggplot2 Code of Conduct, which used a much earlier version of this document, writes:
> As contributors and maintainers of this project, ...

Maybe that's better?

Reply:
I think so, it sounds clearer to me.